.github/workflows/push-collapse-query-script.yml

```
@@ -0,0 +1,53 @@
+name: Create Clickhouse Search Query Collapsing Script
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+    paths:
+    - 'docker/collapse-query-script/**'
+
+jobs:
+  build_collapse_query_script:
+    name: Push Collapse Query Script
+    runs-on: blacksmith-2vcpu-ubuntu-2204
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v4
+
+      # - name: Set up QEMU
+      #   uses: docker/setup-qemu-action@v3
+
+      - name: Setup buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            trieve/clickhouse-collapse-query-script
+          tags: |
+            type=raw,latest
```

```suggestion
            type=raw,latest
            type=sha
```
Add this so we can tag the releases to the git commit